tests/src/services/users.service.test.ts

```diff
@@ -99,8 +132,8 @@ describe('UsersService tests', () => {
     });
   });
 
-  describe('Find customer by User UUId', () => {
-    it('When looking for a customer by UUID with the correct params, then the customer is found', async () => {
+  describe('findUserByUuid()', () => {
```

Comment:
This goes directly against what we talked about in the last review. Talk about the use case, not about the function literally